### PR TITLE
High Contrast Theme

### PR DESCRIPTION
**Pull request by CorruptedCas**
image

### CODE REVIEW

File: src/theme/css/variables.css [hunk 1]
```diff
@@ -293,6 +293,46 @@
     
     --chapter-hover-bg: var(--theme-hover);
 }
+.highcon {
```
Comment (priyanujgogoi-28) [inline src/theme/css/variables.css]:
```suggestion

/* High Contrast */
.highcon {
```

File: src/theme/css/variables.css [hunk 2]
```diff
@@ -293,6 +293,46 @@
...
+
+    --searchbar-border-color: #04afff;
+    --searchbar-bg: #070707;
+    --searchbar-fg: #ffd334;
+    --searchbar-shadow-color: #ffffff;
+    --searchresults-header-fg: #ffffff;
+    --searchresults-border-color: #ffffff;
+    --searchresults-li-bg: #fff;
+    --search-mark-bg: #fff;
+}
```
Comment (priyanujgogoi-28) [inline src/theme/css/variables.css]:
```suggestion
}
/* High Contrast */
```

File: src/theme/css/variables.css [hunk 3]
```diff
@@ -293,6 +293,46 @@
...
+
```
Comment (priyanujgogoi-28) [inline src/theme/css/variables.css]:
```suggestion
```

**Review by priyanujgogoi-28 — CHANGES_REQUESTED** [review]
Hello! CorruptedCas,
You have added `High Contrast` theme in wrong section. It should be placed in `Community Themes` and not in `Initial Themes` section.

**Review by MineBartekSA — CHANGES_REQUESTED** [review]
Please address issues and changes suggested by @Priyanuj-Gogoi

**Comment by CorruptedCas** [comment]
Oops lol

**Review by noituri — APPROVED** [review]

File: src/theme/css/variables.css [hunk 4]
```diff
@@ -339,6 +339,50 @@
...
+/* High Contrast */
+
```
Comment (priyanujgogoi-28) [inline src/theme/css/variables.css]:
```suggestion
```

File: src/theme/css/variables.css [hunk 5]
```diff
@@ -339,6 +339,50 @@
...
+    --searchbar-border-color: #04afff;
+    --searchbar-bg: #070707;
+    --searchbar-fg: #ffd334;
+    --searchbar-shadow-color: #ffffff;
+    --searchresults-header-fg: #ffffff;
+    --searchresults-border-color: #ffffff;
+    --searchresults-li-bg: #fff;
+    --search-mark-bg: #fff;
+}
+
```
Comment (priyanujgogoi-28) [inline src/theme/css/variables.css]:
```suggestion
```

**Review by priyanujgogoi-28 — CHANGES_REQUESTED** [review]

**Review by priyanujgogoi-28 — APPROVED** [review]
Thanks for your contribution!

**Review by MineBartekSA — APPROVED** [review]

**Pull request merged**